include/dd/ComplexTable.hpp

```diff
@@ -334,9 +350,14 @@ namespace dd {
             count     = 0;
             peakCount = 0;
 
-            collisions = 0;
-            hits       = 0;
-            lookups    = 0;
+            collisions        = 0;
+            insert_collisions = 0;
```

Comment:
I'd suggest to stick to one naming pattern throughout the project as much as possible.
consider using camelCase.
```suggestion
            insertCollisions = 0;
```